src/test/java/pl/touk/sputnik/processor/detekt/DetektProcessorTest.java

```diff
@@ -1,22 +1,17 @@
 package pl.touk.sputnik.processor.detekt;
 
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 import pl.touk.sputnik.configuration.Configuration;
 import pl.touk.sputnik.configuration.ConfigurationBuilder;
-import pl.touk.sputnik.review.Review;
-import pl.touk.sputnik.review.ReviewFile;
-import pl.touk.sputnik.review.ReviewFormatterFactory;
-import pl.touk.sputnik.review.ReviewResult;
-import pl.touk.sputnik.review.Severity;
-import pl.touk.sputnik.review.Violation;
+import pl.touk.sputnik.review.*;
```

Comment:
i won't write more comments for star imports... :)

Reply:
Done